generators/profilegen/src/defs/definition.rb

```diff
@@ -0,0 +1,24 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
```

Comment:
could you provide more context/explanation on the files you're adding for profile generation?